examples/protocols/multicolor-particle-calibration/artifacts/multicolor-particle-calibration.md

```diff
@@ -45,18 +45,18 @@ Adapted from [https://dx.doi.org/10.17504/protocols.io.bht7j6rn](https://dx.doi.
+24. Perform a series of 10 2-fold dilutions on A1 using `Phosphate Buffered Saline` as diluent to a final volume of 100.0 microliter in wells A1:A11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
 
 ![](../figures/serial_dilution.png)
 <p align="center">Serial Dilution</p>
 
-25. Perform a series of 10 2-fold dilutions on B1 using `Phosphate Buffered Saline` as diluent to a final volume of 200.0 microliter in wells B1:B11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-26. Perform a series of 10 2-fold dilutions on C1 using `Phosphate Buffered Saline` as diluent to a final volume of 200.0 microliter in wells C1:C11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-27. Perform a series of 10 2-fold dilutions on D1 using `Phosphate Buffered Saline` as diluent to a final volume of 200.0 microliter in wells D1:D11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-28. Perform a series of 10 2-fold dilutions on E1 using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in wells E1:E11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-29. Perform a series of 10 2-fold dilutions on F1 using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in wells F1:F11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-30. Perform a series of 10 2-fold dilutions on G1 using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in wells G1:G11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-31. Perform a series of 10 2-fold dilutions on H1 using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in wells H1:H11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
+25. Perform a series of 10 2-fold dilutions on B1 using `Phosphate Buffered Saline` as diluent to a final volume of 100.0 microliter in wells B1:B11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
```

Comment:
This volume change from 200 ul -> 100 ul is not correct.  When a serial dilution is performed, wells A1:A10 retain 100 ul of sample, while the very last well in series, A11, will have a "final volume of 200 ul".  That is why Step 32 then tells the operator to discard 100 ul from wells in column 11, normalizing all volumes to 100 ul.

Note this wording was vetted by the iGEM interlab group, including Jacky Loo, Traci Haddock, and Alejandro Vignoni